src/clarity-angular/layout/tabs/tab-link.directive.ts

```diff
@@ -10,26 +10,26 @@ import {
     HostListener,
     Inject,
     Input,
+    Renderer2,
     ViewContainerRef
 } from "@angular/core";
+import {Subscription} from "rxjs/Subscription";
+
 import {IF_ACTIVE_ID, IfActiveService} from "../../utils/conditional/if-active.service";
 import {TemplateRefContainer} from "../../utils/template-ref/template-ref-container";
+
 import {AriaService} from "./aria-service";
 
 let nbTabLinkComponents: number = 0;
 
 @Directive({
     selector: "[clrTabLink]",
     host: {
-        "[id]": "tabLinkId",
-        "[attr.aria-selected]": "active",
-        "[attr.aria-controls]": "ariaControls",
         "role": "presentation",
```

Comment:
If you've replaced some host bindings, would it be worthwhile to replace them all?

Reply:
I don't see any compelling reasons to replace all of them for now. 🤔  The replaced host bindings were the ones that were holding dynamic values so I needed to use Renderer to set them in a certain lifecycle hook.